### PR TITLE
Typings for material-ui-datatables npm package

### PR DESCRIPTION
**Pull request by ravihugo**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
@ravihugo Please fix the failures indicated in the Travis CI log.

**Comment by andy-ms** [comment]
Please rebase against master; #21156 fixes tests

**Comment by ravihugo** [comment]
@RyanCavanaugh I think I got through all the errors I understood today, but this latest error has got me confused.

**Comment by ravihugo** [comment]
Alright, I think I am done making changes to the PR. Sorry was first time submitting and had a lot of learning on the way :)

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

File: types/args/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// Type definitions for args 3.0
+// Project: https://github.com/leo/args#readme
+// Definitions by: Slessi <https://github.com/Slessi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const c: args;
+export = c;
+
+interface args {
```
Comment (andy-ms) [inline types/args/index.d.ts]: You could just directly export `const`s and `function`s instead of `export =`-ing an object.

**Review by ghost — COMMENTED** [review]
This PR seems to have a few unrelated changes, could you rebase to include just your changes? (Be sure to back up first as rebasing can destroy data)

**Comment by ravihugo** [comment]
Ok did the rebase correctly this time I think.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**